src/reducers/util.js

```diff
@@ -35,6 +35,10 @@ export function updateMedia (obj, type, url) {
   return {...obj, media}
 }
 
+const isSimpleArray = array => isArray(array) && (array.length === 0 || !isObject(array[0]))
+
+const mergeArray = (dest, src) => uniq(dest.concat(src))
```

Comment:
the logic behind using `uniq` here seems way too specific to pinning to fold into mergeList. it seems like it would lead to subtle & difficult-to-diagnose bugs. The underlying assumption you're enforcing here is that any time you have an array of numbers, they are ID's and so you want them to be unique -- that seems like an awkward thing to impose upon everyone who uses or might want to use `mergeList`